relayer-cli/src/devnetRelayExample.ts

```diff
@@ -20,11 +24,11 @@
 (async () => {
   while (1) {
     for (const chain_id of chain_ids) {
-      let nonce = await initialize(chain_id);
+      let nonce = await initialize(chain_id, network);
       // This is libghtbulb switch address in arbitrum sepolia
       const sender = "0x28d6D503F4c5734cD926E96b63C61527d975B382";
```

Comment:
Ok, will consider coderabbitai's reviews as regular reviews and act accordingly 👍.

I am not sure if we plan to deploy the relayer for devnets, which is what this file is intended for. If we do, it would be better to extract the sender address. Otherwise I thought it would be populating the env file.